### PR TITLE
(DE28747) Updated empty states

### PR DESCRIPTION
**Pull request by trevorjonesd2l**
Old and New refer to the updated flag being on or off

---

User does not have any courses pinned: (Old)
> You don’t have any pinned courses. View All Courses to browse all available courses.
 
User does not have any courses: (Old and New)
> You don't have any courses to display. If you believe this is an error, please contact your administrator.
 
User does not have any current of future courses: (New)
> You don't have any current or future courses. View All Courses to browse your past courses.

### CODE REVIEW

File: src/d2l-my-courses-content/d2l-my-courses-content-animated.html [hunk 1]
```diff
@@ -51,11 +51,6 @@
 			<template is="dom-repeat" items="[[_alerts]]">
 				<d2l-alert type="[[item.alertType]]">
 					{{item.alertMessage}}
-					<a
-						is="d2l-link"
-						href="Javascript:void(0);"
-						hidden$="[[_hasEnrollments]]"
-						on-tap="_refreshPage">{{localize('refresh')}}</a>
```
Comment (ryantmer) [inline src/d2l-my-courses-content/d2l-my-courses-content-animated.html]: Both the `_refreshPage` handler and the `refresh` langterm can be removed now (although keeping the langterm around doesn't seem like the end of the world)

File: src/d2l-my-courses-content/d2l-my-courses-content-behavior.html [hunk 2]
```diff
@@ -545,6 +547,13 @@
 		},
 		_refreshPage: function() {
 			document.location.reload(true);
+		},
+		_addOnlyPastCoursesAlert: function(hasCurrentOrFutureCourseArg, hidesPastCoursesArg) {
```
Comment (ryantmer) [inline src/d2l-my-courses-content/d2l-my-courses-content-behavior.html]: The args aren't ever passed in to this function :octocat:

Comment (trevorjonesd2l) [reply src/d2l-my-courses-content/d2l-my-courses-content-behavior.html]: They once were but I ended up keeping them for testing... The css selectors became to annoying to stub in this context

File: src/d2l-my-courses-content/d2l-my-courses-content-animated.html (hunk 1 above)
Comment (trevorjonesd2l) [reply src/d2l-my-courses-content/d2l-my-courses-content-animated.html]: Might as well remove it

File: src/d2l-my-courses-content/d2l-my-courses-content-behavior.html (hunk 2 above)
Comment (ryantmer) [reply src/d2l-my-courses-content/d2l-my-courses-content-behavior.html]: Hmmm, I see I see. Yeah kinda gross to keep them for just testing, but if there's not really another way, I getchu.

File: lang/nb.html [hunk 3]
```diff
@@ -85,6 +84,8 @@
 			'overlay.courseClosedOn': 'Closed {dateTime}',
 			'overlay.courseOpens': 'Opens On',
 			'overlay.courseOpeningOn': 'Opens on {dateTime}',
```
Comment (ryantmer) [inline lang/nb.html]: Is this file missing the `coursePinButton` langterm? 😕

**Pull request merged**